src/source/PeerConnection/JitterBuffer.c

```diff
@@ -221,12 +221,16 @@ STATUS jitterBufferInternalParse(PJitterBuffer pJitterBuffer, BOOL bufferClosed)
         } else {
             lastNonNullIndex = index;
             retStatus = hashTableGet(pJitterBuffer->pPkgBufferHashTable, index, &hashValue);
-            pCurPacket = (PRtpPacket) hashValue;
-            if (retStatus == STATUS_SUCCESS || retStatus == STATUS_HASH_KEY_NOT_PRESENT) {
+            if (retStatus == STATUS_SUCCESS) {
                 retStatus = STATUS_SUCCESS;
```

Comment:
This is a no-op. Perhaps you need the following

CHK(retStatus == STATUS_SUCCESS || retStatus == STATUS_HASH_KEY_NOT_PRESENT, retStatus);
if (retStatus == STATUS_HASH_KEY_NOT_PRESENT) {
    retStatus = STATUS_SUCCESS;
} else {
    // Do the rest of the function. Try to avoid loop controls within a larger loops such as break; and continue; or restructure the loop
}